ivy/functional/ivy/data_type.py

```diff
@@ -1963,7 +1963,35 @@ def type_promote_arrays(
 
 
 def unset_default_dtype():
-    """"""
+    """
+    Unsets the datatype dtype from default data type.
+
+    Parameters
+    ----------
+    None
+
+    Examples
+    --------
+
```

Comment:
There shouldn't be a blank line here according to numpy style docstring test

Reply:
Ok, I will change it.

Reply:
@jkeane508 made suggested changes. Please let me know if any other changes need to be done.

Reply:
Yea this looks good to me!